app/modules/users/models.py

```diff
@@ -624,31 +624,36 @@ def get_asset_groups(self):
         return GitStore.filter_for(AssetGroup, self.git_stores)
 
     @module_required('asset_groups', resolve='warn', default=[])
-    def unprocessed_asset_groups(self):
+    def get_unprocessed_asset_groups(self):
```

Comment:
If this is only used locally, it shoudl really have had an _ at the start of the method to indicate that it was private.

Reply:
gotcha.  i am not wholly unconvinced it _might_ be useful elsewhere.  but thats a wild guess basically.